### PR TITLE
Fixed markdown formatting issue

### PR DESCRIPTION
**Pull request by RishabhTayal**
Fixed markdown formatting issue when Fastfile and Appfile are present

### CODE REVIEW

File: fastlane/lib/fastlane/actions/gym.rb [hunk 1]
```diff
@@ -60,15 +60,26 @@ def self.run(values)
           else
             self.show_xcode_9_warning
           end
+        elsif Actions.lane_context[SharedValues::SIGH_PROFILE_PATHS]
```
Comment (RishabhTayal) [inline fastlane/lib/fastlane/actions/gym.rb]: I am not sure how these changes got added. My intention is to add change only to  [environment_printer.rb](https://github.com/fastlane/fastlane/pull/10284/files#diff-904c2f45a1357fe85cc1c98aeb122e08)

**Comment by KrauseFx** [comment]
Looks great, thanks for the PR, please remove the changes you didn't want to do, and we're good to go

**Comment by RishabhTayal** [comment]
@KrauseFx Do you know how and why those changes got introduced? I didn't any changes other than environment_printer.rb

**Comment by RishabhTayal** [comment]
Updated the PR. I think it's ready to review now.

**Review by KrauseFx — APPROVED** [review]
Yeah, thanks :+1:

**Pull request merged**